docs/reference/migration/migrate_6_0/mappings.asciidoc

```diff
@@ -1,6 +1,15 @@
 [[breaking_60_mappings_changes]]
 === Mapping changes
 
+==== The `_all` meta field is now disabled by default
+
+On new mappings, the `_all` meta field that contains a copy of the text from
+each field is not disabled by default. The `query_string` and
+`simple_query_string` queries that previously used `_all` to search will now
+check if `_all` is enabled/disabled and switch to executing the query across all
+fields if `_all` is disabled. `_all` may be re-enabled by setting `enabled`:
+`true` in the mapping configuration for `_all`.
+
```

Comment:
Maybe add a sentence that says that a reindex is required when `_all` is re-enabled ?